### PR TITLE
[docs minor]space for code-block

### PR DESCRIPTION
**Pull request by ringabout**
image
to
image

After Pr:
image

### CODE REVIEW

File: compiler/sigmatch.nim [hunk 1]
```diff
@@ -568,7 +568,7 @@ proc inconsistentVarTypes(f, a: PType): bool {.inline.} =
 
 proc procParamTypeRel(c: var TCandidate, f, a: PType): TTypeRelation =
   ## For example we have:
-  ##..code-block:: nim
+  ##.. code-block:: nim
```
Comment (timotheecour) [inline compiler/sigmatch.nim]: still incorrect; `nimlint` should report this.

this works (rst needs awkard newlines/spacings):
```
  ## For example we have:
  ##
  ## .. code-block:: nim
  ##   proc myMap[T,S](sIn: seq[T], f: proc(x: T): S): seq[S] = ...
  ##   proc innerProc[Q,W](q: Q): W = ...
```

as can be seen by exporting the symbol and `nim doc -r compiler/sigmatch.nim`, or simply by running `nim doc -r --docInternal compiler/sigmatch.nim`

yes, `--docInternal` is nifty.

Ideally though, `code-block:: nim` should all be replaced by runnableExamples.

Comment (ringabout) [reply compiler/sigmatch.nim]: Indeed.

File: compiler/sigmatch.nim [hunk 2]
```diff
@@ -568,7 +568,7 @@ proc inconsistentVarTypes(f, a: PType): bool {.inline.} =
 
 proc procParamTypeRel(c: var TCandidate, f, a: PType): TTypeRelation =
   ## For example we have:
-  ##..code-block:: nim
+  ## .. code-block:: nim
```
Comment (timotheecour) [inline compiler/sigmatch.nim:572]: still not correct, see my previous comment: try `nim doc -r --docInternal compiler/sigmatch.nim`

Comment (ringabout) [reply compiler/sigmatch.nim:572]: Thanks! I got it.

**Review by timotheecour — APPROVED** [review]

**Pull request merged**